git/homework.py

```diff
@@ -10,28 +10,37 @@ class OurAwesomeException(Exception):
     pass
 
 
-def is_two_object_has_same_value(first: Any, second: Any) -> bool:
+def is_two_object_has_same_value(first_value: Any, second_value: Any) -> bool:
     """
     If @first and @second has same value should return True
     In another case should return False
     """
-    pass
+    if first_value == second_value:
+        return True
+    else:
+        return False
 
```

Comment:
redundant operator if-else. operator "==" already returns bool value.
Try to simplify this code.